pandas-stubs/core/indexes/multi.pyi

```diff
@@ -68,6 +69,8 @@ class MultiIndex(Index):
     def __contains__(self, key) -> bool: ...
     @property
     def dtype(self) -> np.dtype: ...
+    @property
+    def dtypes(self) -> pd.Series[Any]: ...
```

Comment:
Could probably be tightened to `pd.Series[numpy.dtype | pd.api.extensions.ExtensionDtype]` but that probably causes trouble (I don't think those classes are part of S1)

Reply:
With `pd.Series[numpy.dtype | pd.api.extensions.ExtensionDtype]` mypy says

```
pandas-stubs/core/indexes/multi.pyi:73: error: Invalid type argument value for "Series"  [type-var]
tests/test_indexes.py:862: error: Invalid type argument value for "Series"  [type-var]
```